.github/workflows/main.yml

```
@@ -128,6 +119,18 @@ jobs:
                   HASHER_API_AZ_TENANT_ID: ${{ secrets.EXPORT_AZ_TENANT_ID }}
                   HASHER_API_AZ_KEY_VAULT_NAME: ${{ secrets.EXPORT_AZ_KEY_VAULT_NAME }}
               run: |
+                  # We need PIXL_USER_UID and PIXL_USER_GID from the test env file.
+                  source .env
+                  # Export API (running as pixl) needs to be able to *read* files in
+                  # the export dir, but it doesn't write there.
+                  # So, set ownership/permissions accordingly.
+                  # Bear in mind that ownership/perms of files created below this dir will
+                  # depend on the CLI process. I think we don't need to set ACLs for now.
```

```suggestion
                  # Export API (running as pixl) needs only *read* files in the export dir.
                  # Ownership/perms of files created in the export dir will
                  # depend on the user running the CLI process.
```

Not adamant about the wording but would remove the opinions/questions and make it shorter and more to the point